### PR TITLE
Add funky to powershell base image, fix tests

### PR DESCRIPTION
**Pull request by tenczar**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,11 +15,12 @@ RUN mv /powershell/PSDepend/0.1.64/PSDependScripts/Noop.ps1 /powershell/PSDepend
 ## base image
 FROM vmware/photon2:20180424
 
-RUN tdnf install -y powershell-6.0.1-1.ph2
+RUN tdnf install -y powershell-6.0.1-1.ph2 gzip tar
```
Comment (rjew) [inline Dockerfile:18]: Is gzip required here?

File: Dockerfile [hunk 2]
```diff
@@ -29,13 +30,16 @@ COPY function-template ${FUNCTION_TEMPLATE}/
...
 
-ENV WORKDIR=/root/function PORT=8080
+ENV WORKDIR=/root/function PORT=8080 SERVERS=$servers
 EXPOSE ${PORT}
 WORKDIR ${WORKDIR}
 
 COPY ./index.ps1 /root
 
+RUN curl -L https://github.com/dispatchframework/funky/releases/download/0.1.1/funky0.1.1.linux-amd64.tgz -o funky0.1.1.linux-amd64.tgz
+RUN tar -xzf funky0.1.1.linux-amd64.tgz
```
Comment (rjew) [inline Dockerfile]: Should probably store the funky version in a variable so its easy to change in the future.

File: index.ps1 [hunk 3]
```diff
@@ -116,6 +86,9 @@ if ($MyInvocation.Line.Trim() -notmatch '^\.\s+') {
         } else {
             $r = processRequest $request
 
+            if ($r.ContainsKey('stacktrace')) {
+                $response.StatusCode=500
```
Comment (rjew) [inline index.ps1]: Is it possible that the function could return a valid result that contains `stacktrace`? We wouldn't want to return a 500 unless an exception occured.

File: Dockerfile [hunk 4]
```diff
@@ -15,11 +15,12 @@ RUN mv /powershell/PSDepend/0.1.64/PSDependScripts/Noop.ps1 /powershell/PSDepend
 ## base image
```
Comment (rjew) [inline Dockerfile:15]: Should bump the image version in `version.txt`

**Review by rjew — COMMENTED** [review]
Looks good, just a few comments

File: Dockerfile (hunk 1 above)
Comment (tenczar) [reply Dockerfile:18]: Yes. the tarball is compressed with gzip. I'm surprised that installing tar didn't include gzip as a dependency.

File: Dockerfile (hunk 2 above)
Comment (tenczar) [reply Dockerfile]: good call

File: Dockerfile (hunk 4 above)
Comment (tenczar) [reply Dockerfile:15]: oops, will do

**Review by rjew — APPROVED** [review]

**Pull request merged**